api/health.py

```diff
@@ -0,0 +1,17 @@
+import requests
+from flask import Blueprint, current_app
+
+from api.utils import jsonify_errors, jsonify_data
+
+health_api = Blueprint('health', __name__)
+
+
+@health_api.route('/health', methods=['POST'])
+def health():
+    response = requests.get(current_app.config['API_URL'])
+
+    if response.ok:
+        return jsonify_data({'status': 'ok'})
+
+    error = response.json()['error']
+    return jsonify_errors({'errors': [error]})
```

Comment:
I think this will not work correctly because:

def jsonify_errors(error):
    error['code'] = error.pop('status').lower()
    error.pop('details', None)
    error['type'] = 'fatal'
    return jsonify({'errors': [error]})

Reply:
Right) Thanks